### DISCUSSION TITLE
How To Make A Block Work With A Beacon

### DISCUSSION THREAD

**Discussion by mangodev01** [FabricMC/fabric-api#2432] [Mod Dev Support] [answered]
Hi, I wanted to make a ruby block in my mod (1.19), and I want to make it work with beacons, so you can build a beacon pyramid from this block, but I didn't find any methods so I can do this in my ModBlocks

**Comment by apple502j** [comment, marked as answer]
There is a tag for that: `minecraft:beacon_base_blocks`

**Reply by mangodev01** [reply]
thanks for the answer, also do you know how to make an item power a beacon, is there another tag  for it or something?

**Reply by OroArmor** [reply]
Yes, there is another tag for that as well. `minecraft:beacon_payment_items`

**Reply by mangodev01** [reply]
_**thanks**_

**Comment by mangodev01** [comment]
On Sat, 30 Jul 2022 at 14:08, apple502j ***@***.***> wrote:

> There is a tag for that: minecraft:beacon_base_blocks
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/orgs/FabricMC/discussions/2432#discussioncomment-3286416>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AZLKOVFYSGDU6G57SWZQCTLVWUELHANCNFSM55DAM5NA>
> .
> You are receiving this because you authored the thread.Message ID:
> ***@***.***>
>
Thanks, i will try making this tab for my block
